### ISSUE TITLE
Layout changes after forking new version. 

### ISSUE THREAD

**Issue by TMorville** [mmistakes/minimal-mistakes#1451]
- [ x] This is a question about using the theme.
- [ ] This is a feature request.
- [ ] I believe this to be a bug with the theme.
  - [x ] I have updated all gems with `bundle update`.
  - [x ] I have tested locally with `bundle exec jekyll build`.

---

## Environment informations

- **Minimal Mistakes version: 4.9.0** 
- **Jekyll version: 3.7.0**
- **GitHub Pages hosted: yes** 
- **Operating system: osx Sierra** 

---

** This is a support question ** 

---
I have a [github page][1] up and running using Jekyll with the [Minimal Mistakes theme][2]. As I had some ruby problems when serving I decided to reinstall & update the entire thing by merging with the newest repo. Since my changes had been minimal, it was easy to implement my personal pages etc.

However, for some reason there are some quite noticeable changes to the layout. 

**First**, the header is now right-aligned:
[![enter image description here][3]][3]

Versus left (or centered?), which is what I would ideally want:
[![enter image description here][4]][4]

**Second**, each column using the "Single" layout seems wider: 

[![enter image description here][5]][5]

Versus this, more narrow, single column:

[![enter image description here][6]][6]

I am pretty much a CSS/HTML ignoramus, so I definitely have not changed those to be left aligned & more narrow when I deployed the site. How can I change these things? So far, I've tried looking through header & masthead in _included, but without any idea what custom code snippets I could add to 1) make the header left aligned or centered 2) make each page with the "Single" layout more narrow. 

Thanks!

  [1]: https://tmorville.github.io/
  [2]: https://github.com/mmistakes/minimal-mistakes
  [3]: https://i.stack.imgur.com/nZTIk.png
  [4]: https://i.stack.imgur.com/kZ2aR.png
  [5]: https://i.stack.imgur.com/fgzh8.png
  [6]: https://i.stack.imgur.com/YOJmR.png

**Comment by mmistakes** [comment]
Doesn't look like you have the latest version of the theme. See the [upgrade docs](https://mmistakes.github.io/minimal-mistakes/docs/upgrading/) for more info on how to determine that.

There have been numerous enhancements and changes related to aligning the main and sidebar columns. You likely have a version much older than all that, which is why you're seeing some inconsistencies. That coupled with you not having the latest HTML and CSS is breaking the layout.

Also it might be worth using the [remote theme option](https://mmistakes.github.io/minimal-mistakes/docs/quick-start-guide/#github-pages-method) instead of forking since it's much easier to upgrade. You won't have to ever mess merging in commits from this repo.

**Comment by TMorville** [comment]
I've implemented the remote theme option and manually resolved the issues. Now if I check my /_sass/minimal-mistakes.scss file it says 

```
/*!
 * Minimal Mistakes Jekyll Theme 4.9.0 by Michael Rose
 * Copyright 2013-2018 Michael Rose - mademistakes.com | @mmistakes
 * Licensed under MIT (https://github.com/mmistakes/minimal-mistakes/blob/master/LICENSE.txt)
*/
```

So it looks like the update worked. 

However, the layout discrepancies remain, and a new one occurred. E.g. the pages are "wide" and the header is right aligned. Further, my projects page does not seem to render correctly. Under _pages I have a file called projects.html which contains

```
---
layout: archive
title: projects
permalink: /projects/
author_profile: false
---

{% for post in site.finprojects %}
  {% include archive-single.html %}
{% endfor %}

<h2><i>work in progress below</i></h2>
<hr>

{% for post in site.projects %}
  {% include archive-single.html %}
{% endfor %}
```

But the different .md posts are not shown. 

To summarise: 

1) How can I move my header to center or left?
2) Can I decrease the standard "width" of the single column layout?
3) Is there something wrong with my projects.html file?

I appreciate that you don't have time to answer custom layout questions. Any reference to where I would be able to learn this would be appreciated.

**Comment by mmistakes** [comment]
Short answer, yes the header was right aligned at some point as were adjusting the column widths of the sidebar/main to accommodate the TOC better.

I did adjust it several releases ago so you could sift through old commits and see what changed. You could also search through old issues with keywords like "grid", "width", "layout", etc. I've given many detailed answers to others explaining how to go about adjusting the layout via CSS.

Also your browser's development tools are your friend. You can inspect elements and view the CSS to tinker with in browser before changing the actual source files.
